src/browser/base/content/ZenUIManager.mjs

```diff
@@ -768,3 +770,252 @@ var gZenVerticalTabsManager = {
     this._tabEdited = null;
   },
 };
+
+var gZenMediaController = {
```

Comment:
Only thing I would suggest is adding this class into a different file

Reply:
You want me to create a new file?

Reply:
That would be great yeah, ZenMediaControler.mjs or something